### DISCUSSION TITLE
appendix in pdf

### DISCUSSION THREAD

**Discussion by ygalanak** [quarto-dev/quarto-cli#3943] [Q&A]
Hi,

Is there a way to include an appendix when using quarto to produce a .pdf? In that case, tables and figures should have the caption: "table A1."  and "Figure A1." respectively.

**Comment by cderv** [comment]
Appendix is not part of the default Pandoc template. This would require a custom format template to handle appendix. Though I am not sure that cross reference will get the numbering right. 

Example of custom format for PDF are in https://quarto.org/docs/extensions/listing-journals.html - Some of them already have appendix. 

@dragonstyle we don't deal specifically of appendix in our numbering yet for the cross references in Quarto right ?

**Reply by dragonstyle** [reply]
This is well supported for PDF/LaTeX books, but not individual articles afaik. It would be nice if our existing appendix section (for HTML output https://quarto.org/docs/authoring/appendices.html#custom-appendix-sections) could also produce nice LaTeX output.
